deep-security-agent/libraries/dsa.rb

```diff
@@ -22,7 +22,7 @@ def detect_DSA_host_platform
     case node[:platform]
     when /amazon/
       currentDSAPlatform.host_platform = 'amzn'
-      currentDSAPlatform.host_platform_version = '1'
+      currentDSAPlatform.host_platform_version = node[:platform_version] == '2' ? '2' : '1'
```

Comment:
Would you consider to use the platform_version directly? More flexible and possible for future release?

Reply:
On Amazon Linux 2, node[:platform_version] reports 2
However, on Amazon Linux 2, it reports 2018.03, 2017.09, etc

I can't predict the future.